examples/albert/run_first_peer.py

```diff
@@ -163,6 +163,10 @@ def upload_checkpoint(self, current_loss):
                        for peer in metrics_dict]
             latest_step = max(item.step for item in metrics)
             if latest_step != current_step:
+                logger.info(f"Got metrics from {len(metrics)} peers")
+
+                for i in range(len(metrics)):
+                    logger.info(f"{i} peer {metrics[i]}")
```

Comment:
Maybe replace with debug to avoid excessive outputs when everything works as intended?